client/containers/Users/Users.jsx

```diff
@@ -46,8 +46,10 @@ class Users extends Component {
     this.props.fetchUsers('', false, page - 1);
   }
 
-  onSearch = (query) => {
-    this.props.fetchUsers(query);
+  onSearch = (query, filterBy) => {
+    if (query && query.length > 0) {
```

Comment:
What happens if you want to blank out the search bar?  Should we be searching for blank?